go.mod

```diff
@@ -275,5 +275,6 @@ replace (
 	// fix potential security issue(CVE-2020-26160) introduced by indirect dependency.
 	github.com/dgrijalva/jwt-go => github.com/form3tech-oss/jwt-go v3.2.6-0.20210809144907-32ab6a8243d7+incompatible
 	github.com/pingcap/tidb/parser => ./parser
+	github.com/tikv/client-go/v2 => github.com/LykxSassinator/client-go/v2 v2.0.0-20230131091257-ada803e8728e
```

Comment:
Need to update client-go before this PR.

Reply:
CC. Done for it, thx.